include/arch/x86/ffs.h

```diff
@@ -47,7 +47,7 @@ static ALWAYS_INLINE unsigned int find_lsb_set(u32_t op)
 
 	__asm__ volatile (
 
-#if defined(CONFIG_CMOV)
+#if !defined(CONFIG_CPU_MINUTEIA)
```

Comment:
Might be clear enough anyway, but maybe a comment like `/* No CMOV */` could be added somewhere. Maybe the `#ifdef` could be reversed too.

Reply:
Ah, good suggestions.  I've revised the last commit to take them into account.

Reply:
Thx